### PR TITLE
tweak search style

### PR DESCRIPTION
**Pull request by nihgwu**
before:
image

after:
image

### CODE REVIEW

**Comment by joecritch** [comment]
@nihgwu Looking good — thanks! I was meaning to do this.

File: www/src/components/LayoutHeader/Header.js [hunk 1]
```diff
@@ -161,6 +161,7 @@ const Header = ({location}) => (
             css={{
               appearance: 'none',
               background: 'transparent',
+              outline: 'none',
```
Comment (bvaughn) [inline www/src/components/LayoutHeader/Header.js]: I think we should revert this until we have a custom focus style.

Comment (nihgwu) [reply www/src/components/LayoutHeader/Header.js]: there will be a input indicator there when focused, the live jsx editor focus in the same way

Comment (bvaughn) [reply www/src/components/LayoutHeader/Header.js]: Without the `outline` style, there's no visual indicator of focus on the desktop site. There is on the mobile-sized layout, but `outline` is already suppressed there.

Anyway. Thanks for reverting this bit. 😄

**Pull request merged**